v2/internal/pkg/mirror/mirror.go

```diff
@@ -215,7 +223,30 @@ func (o *Mirror) copy(ctx context.Context, src, dest string, opts *CopyOptions)
 			}
 		}
 		return nil
-	}, opts.RetryOpts)
+	}, &retryOpts)
+}
+
+// Custom implementation to extend `containers/common/pkg/retry.retry`
+func isErrorRetryable(err error) bool {
+	var httpError docker.UnexpectedHTTPStatusError
+	switch {
+	case err == nil:
+		return false
```

Comment:
@r4f4 - would it be worth while adding a warning message - indicating to the user that re-try is occurring.  ?

Reply:
I think warning is too much. What about just a debug message? Printing things while the spinners are working messes up the output.

Reply:
The problem is that the retry mechanism happens inside `containers/common` via `retry.IfNecessary` so we don't really have that information unless we reimplement the retry on our side. Notice that `containers/common` already prints log information about retries if `logrus` is properly configured: https://github.com/containers/common/blob/main/pkg/retry/retry.go#L50